packages/v4/patternfly-docs/content/design-guidelines/components/wizard/wizard.md

```diff
@@ -94,7 +95,6 @@ Sub-steps can be added to the sidebar. Use sub-steps when there is a hierarchica
 
 <img src="./img/wizard-with-substeps.png" alt="Wizard with substeps" />
```

Comment:
Wondering if it might make more sense to update this image with a drawer that has some content, rather than using an empty drawer. Especially since the guidelines highlight that the drawer is used when we want to add more content, so it would be helpful to provide an example of that here! WDYT @mmenestr ?